### DISCUSSION TITLE
Switching between Synced and Not Synced

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#2381] [Q&A]
I just upgrade from 1.0.5 to 1.1.1, and the full node status became strange. It randomly switches between Synced and Not Synced. This node runs on Google Cloud Platform, and the connection status is always connected. It could not be the problem of the network. 
What's wrong with that?
Could Chia provide more info on connecting status? Like the nodes interacting with, the IPs, traffic, etc.
Thank you!

**Comment by fabitom** [comment]
Same here, but after about 20 minutes its stable synced. I was also wondering what the cause is.

**Reply by marcosyonaware** [reply]
Same here , after aprox 15 min back to synced
